src/modules/esl-panel/core/esl-panel.ts

```diff
@@ -10,25 +10,37 @@ export interface PanelActionParams extends ToggleableActionParams {
   noCollapse?: boolean;
 }
 
+/**
+ * ESLPanel - is a custom element, that is used as a wrapper for content that
+ * can be shown/hidden or collapsed (animated height change).
```

Comment:
ESLPanel - is a custom element, which is used as a wrapper for content that can be shown/hidden, or collapsed (height animation).

Reply:
updated